src/firefox/index.js

```diff
@@ -248,9 +329,18 @@ export async function useProfile(
   {
     app,
     configureThisProfile = configureProfile,
+    isFirefoxDefaultProfile = isDefaultProfile,
     customPrefs = {},
   }: UseProfileParams = {},
 ): Promise<FirefoxProfile> {
+  const isForbiddenProfile = await isFirefoxDefaultProfile(profilePath);
```

Comment:
Kumar proposed option like `--allow-web-ext-to-completely-break-my-profile`, i think that might be a good idea if we show a warning first, since if users want to do something potentially dangerous that's on them.